lib/modules/home/home_view.dart

```diff
@@ -8,6 +8,8 @@ import 'package:otraku/modules/discover/discover_models.dart';
 import 'package:otraku/modules/discover/discover_providers.dart';
 import 'package:otraku/modules/filter/filter_providers.dart';
 import 'package:otraku/modules/home/home_provider.dart';
+import 'package:otraku/modules/schedule/schedule_provider.dart';
+import 'package:otraku/modules/schedule/schedule_view.dart';
```

Comment:
Ok, so sorry for making you rework this, but I wasn't planning at all to have the schedule as a tab on the home page - here specifically I don't want to have that many tabs. My idea was for the schedule to be a separate page, and the icon button that opens it to be located in the top bar of discover, but only while discover is set to "anime". It can be immediately to the left of the filter icon.
Notably, that means the schedule won't be lazy loaded. There won't be such a flag, and `fetch` would always be called in the constructor of `ScheduleMediaNotifier`.

Reply:
I've put the icon in the bottom bar as a placeholder, so I expected this. 

I'll move it to the discover tab!